main.py

```diff
@@ -51,6 +51,7 @@ def log(message, level="info"):
     "[region]junta[/region]": "Junta",
     "[region]the eternal order[/region]": "TEO",
     "[region]the brothers of auralia[/region]": "TEO",
+    "[region]Lair of the Dragon[/region]": "LotD",
```

Comment:
```suggestion
    "[region]lair of the dragon[/region]": "LotD",
```
WFE criteria should be lowercase to support case-insensitive searching.